docs/administration/helm.md

```diff
@@ -0,0 +1,101 @@
+      NAME: starrocks
+      LAST DEPLOYED: Tue Apr 11 12:11:12 2023
+      NAMESPACE: default
+      STATUS: deployed
+      REVISION: 1
+      NOTES:
+      1. Get the application URL by running these commands:
+        
+      kube-starrocks has been installed, Check its status by running:
+      kubectl --namespace default get starrockscluster -l "cluster=kube-starrocks"
+        
+      Visit https://github.com/starrocks for instructions on how to create & configure.
+      ```
+
+   2. Deploy with custom configurations
+      - Create a YAML file, for example, **my-values.yaml**, and customize the configurations for the StarRocks Operator and StarRocks cluster in the YAML file. For the supported parameters and descriptions, see the comments in the default **[values.yaml](https://github.com/StarRocks/helm-charts/blob/main/charts/kube-starrocks/values.yaml)** of the Helm Chart.
+      - Run the following command to deploy the StarRocks Operator and StarRocks cluster with the custom configurations in **my-values.yaml**.
+
+        ```Bash
+        helm install -f my-values.yaml starrocks starrocks-community/kube-starrocks
+        ```
+
+    Deployment takes a while. During this period, you can check the deployment status by using the prompt command in the returned result of the deployment command above. The default prompt command is as follows:
+
+    ```Bash
+    $ kubectl --namespace default get starrockscluster -l "cluster=kube-starrocks"
+    # If the following result is returned, the deployment has been successfully completed.
+    NAME             FESTATUS   CNSTATUS   BESTATUS
+    kube-starrocks   running               running
+    ```
+
+    You can also run `kubectl get pods` to check the deployment status. If all Pods are in the`Running` state and all containers within the Pods are `READY`, the deployment has been successfully completed.
+
+    ```Bash
+    $ kubectl get pods
+    NAME                                       READY   STATUS    RESTARTS   AGE
+    kube-starrocks-be-0                        1/1     Running   0          2m50s
+    kube-starrocks-fe-0                        1/1     Running   0          4m31s
+    kube-starrocks-operator-69c5c64595-pc7fv   1/1     Running   0          4m50s
+    ```
+
+## Next steps
+
+**Access StarRocks cluster**
+You can access the StarRocks cluster from inside and outside the Kubernetes cluster. For detailed instructions, see [Access StarRocks Cluster](https://chat.openai.com/sr_operator#accessing-the-starrocks-cluster).
+
+> **NOTICE**
+>
+> The StarRocks cluster is located in the namespace `default`.
+
+**Manage StarRocks operator and StarRocks cluster**
+
+- If you need to update the configurations of the StarRocks operator and StarRocks cluster, see [Helm Upgrade](https://helm.sh/docs/helm/helm_upgrade/).
+- If you need to uninstall the StarRocks Operator and StarRocks cluster, run the following command:
+
+  ```bash
+  helm uninstall starrocks
+  ```
```

Comment:
can also add a reference link to our helm chart on artifact hub: https://artifacthub.io/packages/helm/kube-starrocks/kube-starrocks